draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -788,6 +776,8 @@ Multipath QUIC-specific security considerations can be found in Section 8 of {{I
 
 Section 6 of {I-D.ietf-quic-datagram} discusses security considerations specific to the use of the Unreliable Datagram Extension to QUIC.
 
+This document does not explicitly discuss security consideration specific to the use of QUIC as a tunneling protocol towards a ATSSS proxy, as the intention of this document is to provide an informational overview of the on-going work in 3GPP on ATSSS to support non-TCP, rather than discussing a detailed solution.
```

Comment:
```suggestion
This document does not specify specific mechanisms to use QUIC as a tunneling protocol towards a ATSSS proxy, as the intention of this document is to provide an informational overview of the ongoing work in 3GPP on ATSSS to support non-TCP, rather than discussing a detailed solution. Nevertheless, this document cites candidates solutions to provide such tunneling service. Security considerations specific to these solutions are provided below.  
```